tests/playwright/pages/editor-page.js

```diff
@@ -241,6 +276,72 @@ module.exports = class EditorPage extends BasePage {
 	 */
 	async removeFocus() {
 		await this.getPreviewFrame().locator( '#elementor-add-new-section' ).click( { button: 'right' } );
+		await this.getPreviewFrame().locator( '#elementor-add-new-section' ).click();
+	}
+
+	/**
+	 * Hide controls from the video widgets.
+	 *
+	 * @return {Promise<void>}
+	 */
+	async hideVideoControls() {
+		await this.getPreviewFrame().waitForSelector( '.elementor-video' );
+
+		const videoFrame = this.getPreviewFrame().frameLocator( '.elementor-video' ),
+			videoButton = videoFrame.locator( 'button.ytp-large-play-button.ytp-button.ytp-large-play-button-red-bg' ),
+			videoGradient = videoFrame.locator( '.ytp-gradient-top' ),
+			videoTitle = videoFrame.locator( '.ytp-show-cards-title' ),
+			videoBottom = videoFrame.locator( '.ytp-impression-link' );
+
+		await videoButton.evaluate( ( element ) => element.style.opacity = 0 );
+		await videoGradient.evaluate( ( element ) => element.style.opacity = 0 );
+		await videoTitle.evaluate( ( element ) => element.style.opacity = 0 );
+		await videoBottom.evaluate( ( element ) => element.style.opacity = 0 );
+	}
+
+	/**
+	 * Hide controls and overlays on map widgets.
+	 *
+	 * @return {Promise<void>}
+	 */
+	async hideMapControls() {
+		await this.getPreviewFrame().waitForSelector( '.elementor-widget-google_maps iframe' );
+
+		const mapFrame = this.getPreviewFrame().frameLocator( '.elementor-widget-google_maps iframe' ),
+			mapText = mapFrame.locator( '.gm-style iframe + div + div' ),
+			mapInset = mapFrame.locator( 'button.gm-inset-map.gm-inset-light' ),
+			mapControls = mapFrame.locator( '.gmnoprint.gm-bundled-control.gm-bundled-control-on-bottom' );
+
+		await mapText.evaluate( ( element ) => element.style.opacity = 0 );
+		await mapInset.evaluate( ( element ) => element.style.opacity = 0 );
+		await mapControls.evaluate( ( element ) => element.style.opacity = 0 );
+	}
+
+	/**
+	 * Open the page in the Preview mode.
+	 *
+	 * @return {Promise<void>}
+	 */
+	async togglePreviewMode() {
+		if ( ! await this.page.$( 'body.elementor-editor-preview' ) ) {
+			await this.page.locator( '#elementor-mode-switcher' ).click();
+			await this.page.waitForSelector( 'body.elementor-editor-preview' );
+			await this.page.waitForTimeout( 500 );
```

Comment:
@louiswol94 : I needed to make sure the widget panel is hidden, before a screenshot is taken. I tried checking for certain elements to be `.not.toBeVisible()`, but that didn't work.